dev-docs/conventions.md

```diff
@@ -35,43 +35,41 @@ Further we try to adhere to the following guidelines:
   Example:
 
   ```Go
-  log.Infof("This is a log message")
-  log.Infof("Waiting to do something...")
+  log.Info("This is a log message")
+  log.Info("Waiting to do something...")
   log.Error("A critical error occurred!")
   ```
 
-* Use the `With()` method to add structured context to your log messages. The context tags should be easily searchable to allow for easy log filtering. Try to keep consistent tag naming!
+* Use the `WithAttrs()` method to add structured context to your log messages. The context tags should be easily searchable to allow for easy log filtering. Try to keep consistent tag naming!
 
   Example:
 
   ```Go
-  log.With(zap.Error(someError), zap.String("ip", "192.0.2.1")).Errorf("Connecting to IP failed")
+  log.WithAttrs(slog.Any("error" someError), slog.String("ip", "192.0.2.1")).Error("Connecting to IP failed")
   ```
 
-* Log messages may use format strings to produce human readable messages. However, the information should also be present as structured context fields if it might be valuable for debugging purposes.
+* Slog does not support format strings out of the box. You have to wrap any format strings in `fmt.Sprintf()`.
 
   Example:
 
   ```Go
-  log.Infof("Starting server on %s:%s", addr, port)
+  log.Info(fmt.Sprintf("Starting server on %s:%s", addr, port))
   ```
 
-* Usage of the `Fatalf()` method should be constrained to the main package of an application only!
-
 * Use log levels to configure how detailed the logs of you application should be.
 
-  * `Debugf()` for log low level and detailed information. This may include variable dumps, but should not disclose sensitive information, e.g. keys or secret tokens.
-  * `Infof()` for general information.
-  * `Warnf()` for information that may indicate unwanted behavior, but is not an application error. Commonly used by retry loops.
-  * `Errorf()` to log information about any errors that occurred.
-  * `Fatalf()` to log information about any errors that occurred and then exit the program. Should only be used in the main package of an application.
+  * `Debug()` for log low level and detailed information. This may include variable dumps, but should not disclose sensitive information, e.g. keys or secret tokens.
+  * `Info()` for general information.
+  * `Warn()` for information that may indicate unwanted behavior, but is not an application error. Commonly used by retry loops.
+  * `Error()` to log information about any errors that occurred.
+  * `Fatal()` to log information about any errors that occurred and then exit the program. Should only be used in the main package of an application.
 
-* Loggers passed to subpackages of an application may use the `Named()` method for better understanding of where a message originated.
+* Loggers passed to subpackages of an application may use the `WithGroup()` method for better understanding of where a message originated.
```

Comment:
WithGroup doesn't seem the ideal fit for this. Maybe this can better be implemented with a wrapping handler? Didn't look to deep into it. Opinions?

Reply:
It would surely be possible to implement this with a wrapping handler somehow, but I don't really see the benefit. Is there a specific reason why `WithGroup()` isn't ideal?

Reply:
Sorry, nevermind. I only read half of the documentation on slog groups and it seemed to me that the use case is a different one. But they also mention the exact use case we have here.